resources/views/videos/show/content.blade.php

```diff
@@ -23,6 +23,14 @@
         </div>
     </div>
 </div>
+<message-curtain v-if="invalidMoovAtomPosition" v-cloak>
+    <div class="message-curtain--text text-danger">
+        <div class="lead">
+            Please <a href="/manual/tutorials/videos/fix-video-encoding">fix the video encoding</a>.<br>This may cause problems if you continue.
```

Comment:
```suggestion
            The video file contains an invalid moov atom position. This can cause performance issues or incorrect annotation thumbnails in Largo. See <a href="/manual/tutorials/videos/fix-video-encoding">the manual</a> for how to fix this.
```